### PR TITLE
Susom DB on Vert.x 4

### PR DESCRIPTION
**Pull request by glenna-m**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,6 +35,7 @@
   </scm>
 
   <properties>
+    <vertx.version>4.0.0</vertx.version>
```
Comment (garricko) [inline pom.xml:38]: Shouldn't this be 4.1.2?

**Review by garricko — COMMENTED** [review]
Can you also match the existing formatting (e.g. 2 spaces rather than 4)?

File: pom.xml (hunk 1 above)
Comment (glenna-m) [reply pom.xml]: I had been working with 4.0 but am fine with going to the latest.  I have made this change and re-checked it in (squashed).

**Comment by glenna-m** [comment]
Reformatted the logging test with 2 spaces and checked it in (squashed). 

Let me know if there's anything else you'd like.

File: pom.xml [hunk 2]
```diff
@@ -232,6 +233,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-client</artifactId>
+      <version>${vertx.version}</version>
```
Comment (garricko) [inline pom.xml:239]: This should be `<scope>test</scope>`.

**Review by garricko — APPROVED** [review]

**Pull request merged**